### PR TITLE
Personal Portfolio Desktop Version (Template 1)

### PR DESCRIPTION
**Pull request by kessio**
This is a single-page portfolio website with a header, work section, about section, and contact section which was parsed from Microverse Figma Template 1. The webpage is responsive and can be used on different screens. The contact section of the portfolio was done by all the three authors listed below as contributing authors. This project was made with HTML and CSS linters and proper Github Flow was used.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -15,84 +15,896 @@ body {
...
+    justify-content: flex-start;
+    width: 90%;
+    border: #dfe1e6 1px solid;
+    border-radius: 24px;
+    margin-left: 5%;
+    margin-right: 142px;
+    margin-bottom: 142px;
+    padding: 24px;
+    background-color: #fff;
+  }
```
Comment (leonardodiasb) [inline style.css:568]: - [ ] Your work section is not centered on the page. Kindly center this section using `grid` or `flexbox`. You shouldn't center elements using a mix of 'fixed' and '%' margins.

File: index.html [hunk 2]
```diff
@@ -154,25 +246,25 @@ <h2>About <br /> Myself</h2>
                 <div class="resume-btn">Get My Resume</div>
             </a>
```
Comment (leonardodiasb) [inline index.html:247]: - [ ] The 'Get My Resume' is a button and should be inside a proper `<button>` tag. The '<a>' tag around the button has a larger width than the button leading to a bad user experience. Kindly correct the tag name and fix the width.
- [ ] Kindly implement the hover functionalities for this button as well.
image

File: index.html [hunk 3]
```diff
@@ -206,6 +301,7 @@ <h4>Contact Me</h4>
                         <button type="submit" class="contact-btn">Get in Touch</button>
```
Comment (leonardodiasb) [inline index.html:301]: - [ ] Kindly implement the hover functionalities for this button too.

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @kessio , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your README and PR message are descriptive ✔️

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BertrandConxy — APPROVED** [review]
Hi @kessio ✋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💪 
- no linter errors
- Great effort in improvement
- Descriptive PR
- project works fine on my pc

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**